<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterWidget</class>
 <widget class="QWidget" name="FilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>166</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Text label</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filterInternalList">
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>Filter this list</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="filterList"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
